.babelrc

```diff
@@ -1,8 +1,14 @@
 {
-  "plugins": ["transform-object-rest-spread"],
   "presets": [
     [
-      "env"
+      "@babel/preset-env",
+      {
+        "targets": {
+          "node": 14
```

Comment:
How come Node? Generally we want a set of browser targets, for real browser testing. Typically Node is ahead enough anyway that the UI tests will run fine in the Node env if they're transpiled to support the browser runtimes.

Reply:
oh yeah nice catch!

I updated to use `targets: "default"` which translates to `> 0.5%, last 2 versions`.

I think the main goal here is remove polyfill related code, our test env in node / browser could all support native async await without regenerator runtime

With that said, the babel only does one thing: transform esm to commonjs.